### PR TITLE
(docs) update README.md

### PR DESCRIPTION
**Pull request by vladdoster**
Changes are focused on increasing readability for non-native English speakers.

- correct grammar
- reduce verbiage
- update code fences
- consistent header spacing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,42 +35,38 @@ Usage
...
-
 Why the name?
 -------------
 
 <https://en.wikipedia.org/wiki/Geiger_counter>
 
-Unsafe code and ionizing radiation have something in common, they are both
-inevitable in some situations and both should preferably be safely contained!
+Unsafe code and ionizing radiation have something in common, and they are both
+inevitable in some situations, and should preferably be safely contained!
```
Comment (anderejd) [inline README.md:61]: The first added "and" seems out of place when I read the new version as well as the added comma before the last "and". The part after the first comma is means to explain that that have in common. Does that make sense?

**Review by anderejd — CHANGES_REQUESTED** [review]
Looks great thank you! Please see the comment.

File: README.md (hunk 1 above)
Comment (vladdoster) [reply README.md:61]: Unsafe code, like ionizing radiation, is unavoidable in some situations and should be safely contained!

Comment (anderejd) [reply README.md:61]: > Maybe this is too terse? I can just revert this change all together if you want. It just seemed verbose.
>
>> Unsafe code, like ionizing radiation, is unavoidable in some situations and should be safely contained!

Perfect! Let's use that one.

Comment (anderejd) [reply README.md:61]: Merging now and will edit in your suggestion after the merge.

**Pull request merged**